### ISSUE TITLE
Add runtime support for WADL generation

### ISSUE THREAD

**Issue by obreckoff** [quarkusio/quarkus#14725]
**Description**
Functionality to get a WADL representation of the interface of a RESTEasy webservice from url/application.xml.
Without WADL generation functionality it is cumbersome and error-prone to develop clients for large and complex webservices developed with quarkus.

**Implementation ideas**
There is already an implementation in RESTEasy but it seems it has not been imported in quarkus-resteasy yet. 

See: https://docs.jboss.org/resteasy/docs/4.6.0.Final/userguide/html_single/#WADL

The code is located here: https://github.com/resteasy/Resteasy/tree/master/resteasy-wadl

**Comment by obreckoff** [comment]
I built a small proof of concept located at https://github.com/obreckoff/quarkus-with-resteasy-and-wadl-generation/. 

I started from the getting-started-quickstart, added the jaxb- and undertow-extensions and imported the classes from https://github.com/resteasy/Resteasy/tree/master/resteasy-wadl

In these classes I adjusted the logging mechanism to the quarkus conventions because at runtime no implementations of the classes LogMessages.java and Messages.java located in the i18n-package were found. 

The @MessageLogger and @MessageBundle annotations are nowhere used in quarkus and its extensions, so I concluded they have nothing to do with the logging conventions in quarkus. So I deleted the whole i18n-package and copied the messages from Messages.java directly into the code.

Is there any interest to adopt parts of this proof-of-concept in some way into the main branch?
